crates/bevy_render/src/view/window.rs

```diff
@@ -178,18 +178,34 @@ pub fn prepare_windows(
         if window_surfaces.configured_windows.insert(window.id) || window.size_changed {
             render_device.configure_surface(surface, &swap_chain_descriptor);
         }
+        #[cfg(target_os = "linux")]
```

Comment:
This needs a comment right about this cfg statement explaining the context in this PR description.

Reply:
Absolutely necessary, good catch. Thank you for pointing that out.